server/lobbyconnection.py

```diff
@@ -888,6 +888,31 @@ def launch_game(self, game, port, is_host=False, use_map=None):
             else:
                 raise ValueError('invalid type argument')
 
+    async def command_invite_to_party(self, message):
+        invited_player = self.player_service.get_player(message["invited_player_id"])
+        if invited_player is None:
+            raise ClientError("The invited player doesn't exist", recoverable=True)
+        else:
+            self.game_service.team_matchmaking_service.invite_player_to_group(self.player, invited_player)
+
+    async def command_accept_party_invite(self, message):
+        inviting_player = self.player_service.get_player(message["inviting_player_id"])
+        if inviting_player is None:
+            raise ClientError("The inviting player doesn't exist", recoverable=True)
+        else:
+            self.game_service.team_matchmaking_service.accept_invite(self.player, inviting_player)
+
+    async def command_kick_player_from_party(self, message):
+        kicked_player = self.player_service.get_player(message["kicked_player_id"])
+        if kicked_player is None:
+            raise ClientError("The kicked player doesn't exist", recoverable=True)
+        else:
+            self.game_service.team_matchmaking_service.kick_player_from_party(self.player, kicked_player)
+
+    async def command_leave_party(self):
+        self.game_service.team_matchmaking_service.leave_party(self.player)
```

Comment:
I think it would be better to remove some feature envy here and have `LobbyConnection` take `TeamMatchmakingService` in the constructor. Then we can just do `self.team_matchmaking_service...`

Reply:
hmm, yeah, should've done that, gonna move teamMatchmakingService (notice me automatically writing this in camel case :) ) out of game conn to server.py, I was a bit shocked myself back then how ugly this line looked but went on doing sth else first